internal/hcp/registry/hcl.go

```diff
@@ -18,10 +18,11 @@ import (
 
 // HCLRegistry is a HCP handler made for handling HCL configurations
 type HCLRegistry struct {
-	configuration *hcl2template.PackerConfig
-	bucket        *Bucket
-	ui            sdkpacker.Ui
-	metadata      *MetadataStore
+	configuration  *hcl2template.PackerConfig
+	bucket         *Bucket
+	ui             sdkpacker.Ui
+	metadata       *MetadataStore
```

Comment:
Come to think of it, since we change how we get the build names, does that impact that `MetadataStore`? It might not but I just want to be sure we considered delving into this before this gets merged, so that we don't break things later down the line